js/bibox.js

```diff
@@ -1134,6 +1134,62 @@ module.exports = class bibox extends Exchange {
     }
 
     async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        /**
+         * @method
+         * @name bibox#createOrder
+         * @description create a trade order
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market' or 'limit'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float|undefined} price the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
+         * @param {object} params extra parameters specific to the bibox api endpoint
+         * @returns {object} an [order structure]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'side': side,
+            'type': type,
+            'quantity': this.amountToPrecision (symbol, amount),
+            'price': this.priceToPrecision (symbol, price),
+            // 'client_order_id': // Order id, a string with a valid value of an int64 integer
+            // 'time_in_force': // Valid value gtc, ioc
+            // 'post_only':
+        };
+        const clientOrderId = this.safeString2 (params, 'client_order_id', 'clientOrderId');
```

Comment:
remove `client_order_id` and just make it `safeString`. `client_order_id` will end up in the request params anyway

------

Also add `clientOrderId` to the docstring as `params.clientOrderId`